controllers/podchaos/containerkill/types.go

```diff
@@ -0,0 +1,188 @@
+// Copyright 2019 PingCAP, Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package containerkill
+
+import (
+	"context"
+	"errors"
+	"fmt"
+
+	v1 "k8s.io/api/core/v1"
```

Comment:
please group import

Reply:
The result after I executed "goimports -w -l -local github.com/pingcap/chaos-mesh ./" is like this.  Maybe I can adjust it manually. But "goimports" in "makefile" will change back.

Reply:
thx. I have edited it.